src/lib/load-config.ts

```diff
@@ -1,17 +1,17 @@
 import { join as pathJoin } from 'path';
 import { parse as parseYaml } from 'yaml';
+import { readFileSync } from 'fs';
 import { DEFAULT_CONFIG_FILENAME } from './consts';
-import { readFile } from './file';
 import { ConfigTypes } from './types';
 import { createHash } from './hash';
 
-export default async function loadConfig(
+export default function loadConfig(
```

Comment:
I do understand why we want this. So why don't we have a new `loadConfigSync`, so that the existing code can keep being async.

Reply:
Done.